### PR TITLE
Simplify volumes & disks

### PR DESCRIPTION
**Pull request by machacekondra**
Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py [hunk 1]
```diff
@@ -52,12 +52,9 @@
             - List of dictionaries which specify disks of the virtual machine.
             - A disk can be made accessible via four different types: I(disk), I(lun), I(cdrom), I(floppy).
             - All possible configuration options are available in U(https://kubevirt.io/api-reference/master/definitions.html#_v1_disk)
+            - Each disk must have specified a I(volume) that declares which volume type of the disk
```
Comment (pkliczewski) [inline lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py]: W291 trailing whitespace

**Comment by mmazur** [comment]
@machacekondra please correct flake8 complaining and resolve conflicts, then I'll be able to merge.

**Comment by machacekondra** [comment]
Done.

**Pull request merged**